### PR TITLE
Create Payu.ru.xml

### PR DESCRIPTION
**Pull request by ekaris**

### CODE REVIEW

**Comment by J0WI** [comment]
Mismatch on:
(www.)bpp.payu.ru
^payu.ru

Please check, if you can rewrite them to www.
Also note, that our style guide recommends to use tabs instead of spaces.

**Comment by ekaris** [comment]
I didn’t format the rule, the problem must be on make-trivial-rule

**Comment by ekaris** [comment]
Can you tell me how to put the rule right?

File: src/chrome/content/rules/Payu.ru.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+Mismatch on:
+(www.)bpp.payu.ru
+^payu.ru
+-->
+<ruleset name="Payu.ru">
+  <target host="www.payu.ru" />
+  <rule from="^payu.ru"
```
Comment (J0WI) [inline src/chrome/content/rules/Payu.ru.xml]: This should be `^http://payu.ru/`

**Comment by J0WI** [comment]
Please use latest make-trivial-rule from master, this should use tabs.
Also make sure, that if you copy-paste to github.com, you choose tabs as indent mode on the new file form.

File: src/chrome/content/rules/Payu.ru.xml (hunk 1 above)
Comment (ekaris) [inline src/chrome/content/rules/Payu.ru.xml]: Doesn’t work

Comment (J0WI) [inline src/chrome/content/rules/Payu.ru.xml]: You also need to add `<target host="payu.ru" />`

Comment (ekaris) [inline src/chrome/content/rules/Payu.ru.xml]: Doesn’t work

File: src/chrome/content/rules/Payu.ru.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!--
+Mismatch on:
+(www.)bpp.payu.ru
+^payu.ru
+-->
+<ruleset name="Payu.ru">
+	<target host="www.payu.ru" />
+	<target host="payu.ru" />
+	<rule from="^http://payu.ru/"
```
Comment (J0WI) [inline src/chrome/content/rules/Payu.ru.xml]: You need to escape the `.` to `\.`. Just read the errors returned by the Travis tests.

**Comment by ekaris** [comment]
Done

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**